TheBTeam.ConsoleApp/Program.cs

```diff
@@ -1,13 +1,118 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using TheBTeam.BLL;
 
 namespace TheBTeam.ConsoleApp
 {
     class Program
     {
+        enum Category
+        {
+            Home,
+            Education,
+        }
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
-            
+            Person person = new Person();
+
+            var persons = new Dictionary<int, Person>()//default values
+            {
+                 { 1, new Person { FirstName="Santiago", LastName="Castillo", ID="615b4be6281be025752c8800"
+                 ,balance = 2354.13m,currency="PLN",age=32,gender="male",company="RETRACK"
+                 ,email="santiagocastillo@retrack.com",phone="+48 (949) 435-2718"
+                 ,address="490 Poplar Avenue, Skyland, Washington, 3188",registered="2017-03-11T07:27:26 -01:00"} }
+            };
+            foreach (var index in Enumerable.Range(1, 1))
+            {
+                //Console.WriteLine($"Person {index} is {persons[index].FirstName} {persons[index].LastName}");
+            }
+
+            ConsoleKeyInfo keyClick;
+            do
+            {
+                int num;
+                Console.ForegroundColor = ConsoleColor.Green;
+                Console.WriteLine($"Welcome {persons[1].FirstName} {persons[1].LastName} in the financial planner");
+                Console.WriteLine("1. Enter ID user");
+                Console.WriteLine("2. Add new user");
+                Console.WriteLine("3. Enter transation");
+                Console.WriteLine("4. Show transactiosn history for the month");
+                Console.WriteLine("5. Show transaction history for the period");
+                Console.WriteLine("Press key (Esc) to quit!");
```

Comment:
I thing that it would be good way to make manu as Dictionary<int, string> and latter print content via loop
Also i thing we have to rethink our vison of program. Firstly make main menu seen as Admin